### ISSUE TITLE
NTR: Synovial cyst on exam or imaging (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7975]
New term request Synovial cyst on exam or imaging (cohe11_180607160812): Ganglion cyst  UMLS: 0085648

**Comment by pnrobinson** [comment]
@MickeySegal

does this refer to Digital mucous cyst (see https://pubmed.ncbi.nlm.nih.gov/2008470/)?

**Comment by MickeySegal** [comment]
https://www.autoinflammatory-search.org/diseases
"Arthralgias may come before the arthritis. 88% have polyarticular or oligoarticular arthritis, most often in the wrists, knees, and/or ankles. Some with cervical spine, hip, temporomandibular joint arthritis or synovial cysts."

https://www.orthobullets.com/hand/6022/ulnar-tunnel-syndrome?hideLeftMenu=true
"Ulnar Tunnel Syndrome is a compressive neuropathy of the ulnar nerve at the level of the wrist (Guyon's canal), most commonly due to a ganglion cyst."

https://www.ncbi.nlm.nih.gov/books/NBK545200/
"It is also at risk for compression caused by distal radius fracture fragments of by soft tissue masses like lipomas or ganglion cysts"

I think the 2 rheumatologists who were involved with this finding preferred the term synovial cyst; ganglion has the connotation of presence of nerve cells, which is not true for synovial cysts.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000364
- label: Synovial cyst
- parents: [Abnormal synovial bursa morphology (HP:0025231)](https://hpo.jax.org/app/browse/term/HP:0025231)
- pmid: 
- def: A fluid-filled sac or pouch-like mass that is a distention of a preexisting bursa in a joint, tendon sheaths or bursae. Synovial cysts contain synovial fluid and are lined by synovial membrane.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7975
- synonyms:
